src/Scripts/Game/Features/Gathering/EL_GatherAction.c

```diff
@@ -5,31 +5,114 @@ class EL_GatherAction : ScriptedUserAction
 	
 	[Attribute("", UIWidgets.ResourceNamePicker, desc: "Prefab what item is gathered")]
 	private ResourceName m_GatherItemPrefab;
+	
+	[Attribute(defvalue:"1", UIWidgets.EditBox, desc: "Amount of items to receive")]
+	private int m_AmountGathered;
+	
+	[Attribute("", UIWidgets.EditBox, desc: "Display name of what is needed")]
+	private string m_RequiredItemDisplayName;
```

Comment:
This can/should be retrieved from the prefab, so it uses a localized display name of whatever item is being linked. We will need to figure out a good system to cache display names etc for different assets, as to get the info I think we might need to spawn them locally at least once.

Reply:
I can grab the prefab name from inside the script and then format it. As long as all prefabs are named correctly that should work fine.

Reply:
Ah no, was talking about the UIInfo on the inventoryitemcomponent of each of the items. Access to that is easy but would be wasteful to set up every time the action checks it. I will have to do some testing for this before I can give you any reliabe suggestion

Reply:
In the mean time should I remove the Attribute and have it just say "Item Required" until this is figured out. That way other people can start using the updated Gather function?

Reply:
Yup, that is a good idea. I am working on a prefab -> display name cache in the mean time and will add it as soon as its ready :)

Reply:
@Arkensor just updated it to remove that attribute and just default say "Requires item" or "Requires item in hand" for now until the cache is implemented!